generate-client.sh

```diff
@@ -48,11 +48,40 @@ then
   exit 1
 fi
 
+# build npmrc for access to private package repo
+echo "$INPUT_REPO:registry=https://npm.pkg.github.com/" >> .npmrc
+echo "//npm.pkg.github.com/:_authToken=$INPUT_TOKEN" >> .npmrc
+
+# create dummy npm repo and install _current_ version of client package
+mkdir old_version
+cp .npmrc old_version/
+cd old_version
+npm init --yes
+npm i $INPUT_REPO/$INPUT_NAME-client
+cd ..
+
+# diff newly generated js with previous version
+cksum dist/*.js | awk '{print $1":"$2}' > new_checksums
```

Comment:
FYI, these awk statements take the first two fields (with a `:` in between them) from the output of `cksum`. This is required because the third field is the path to each file, which is different for both sets of files. The first two fields of each line are the checksum and the total number of bytes for each file.